reminders/tests.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from django.utils import timezone
+from . import models
+from datetime import datetime
+from events.models import EventParticipant
+from events.tests import new_event, user0  # noqa: F401
+from events.tests import event_participant_creator as participant0  # noqa: F401
+from django.core.exceptions import ValidationError
+from .models import Reminder, time_format
+
+METHOD_0 = models.Reminder.EMethods.EMAIL
+MESSAGES_0 = 'This is a test text.'
+TIME_0 = datetime(2023, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+
+PAST_DATE_TIME_ERROR = "{} is not a valid date_time"
+EXIST_REMINDER_ERROR = 'reminder already exists'
+
+
+@pytest.fixture
+def reminder_0(participant0):  # noqa: F811
+    return models.Reminder(participant_id=participant0, method=METHOD_0, messages=MESSAGES_0, date_time=TIME_0)
+
+
+def test_new_reminder_0(reminder_0, participant0):  # noqa: F811
+    assert reminder_0.participant_id == participant0
+    assert reminder_0.method == METHOD_0
+    assert reminder_0.messages == MESSAGES_0
+    assert reminder_0.date_time == TIME_0
+
+
+@pytest.fixture
+@pytest.mark.django_db()
```

Comment:
instead of repeating the 
@pytest.mark.django_db()
on many test methods you can create a class that wrapped all the tests (lines 18-82) and add it once like:

```suggestion
@pytest.mark.django_db()
class TestReminder():
```